include/graphit/frontend/schedule.h

```diff
@@ -70,6 +70,16 @@ namespace graphit {
 
     };
 
+    struct IntersectionSchedule {
+        enum class IntersectionType {
+            HIROSHI,
```

Comment:
Should also have one that just does binary search?

Reply:
Yeah will add that.